tequilapi/http_routes_test.go

```diff
@@ -19,7 +20,8 @@ func (hfts *handleFunctionTestStruct) httprouterHandle(resp http.ResponseWriter,
 func TestHttpRouterHandlesRequests(t *testing.T) {
 	ts := handleFunctionTestStruct{false}
 
-	router := NewApiEndpoints()
+	idmFake := identity.NewIdentityManagerFake()
+	router := NewApiEndpoints(idmFake)
```

Comment:
You dont need to boo all API routes, while testing only one Router.
Construct `Router{}` structure without factory